setup/install_upstream_rust_keylime/test.sh

```diff
@@ -40,19 +38,9 @@ rlJournalStart
         fi
         #build
         rlRun "cargo build"
```

Comment:
Should we just simply run `make` instead?

Reply:
I'm not sure I just follow previous setup of test.. but I would say that will be better to use `cargo` as specialized tool for rust.

Reply:
Makefile also users cargo. I am thinking that by using `make` we would also test the Makefile itself to some extent. I will update it.

Reply:
Sure, I know that `make` can be used also for rust project I just want to pointed out that `cargo` has a much deeper understanding of Rust.

Reply:
What I mean is that `cargo` is used in the Makefile.
https://github.com/keylime/rust-keylime/blob/master/GNUmakefile#L26